data/human/boarding missions.txt

```diff
@@ -703,3 +703,238 @@ mission "Turn in Tarquin"
 		conversation
 			"You haul captain Tarquin out of your brig and bring him to the portside police station, presenting evidence from your suit cameras and notarized witness statements compiled from your crew members attesting to his crimes of being involved in slavery. The officer on duty thanks you for the service you've rendered, and takes Tarquin into custody. He shoots one final black look at you that sends a shiver up your spine."
 				decline
+
+mission "Boarding Derelict"
```

Comment:
A note for future content creators: if we want to always have a 100% chance for a mission to spawn upon boarding a derelict ship, but we don't want to have to add more and more to this mission's conversation in order to add more derelict ship missions, then we can make this the "default" derelict boarding mission by giving it no `to offer` conditions, and any boarding missions we want to offer aside from this can have `priority` and `to offer` conditions that make them not appear 100% of the time. By giving other missions `priority`, it means we don't need to mess with the alphabetical naming of missions in order to get them to sometimes offer before this one hits its 100% offer chance. Alternatively, we could make this mission `minor` and all other derelict boarding missions non-minor.